tests/acceptance/features/apiSearch1/search.feature

```diff
@@ -92,11 +96,13 @@ Feature: Search
 
   Scenario Outline: user cannot search declined share
     Given using <dav-path-version> DAV path
-    And user "Alice" has created a share inside of space "project101" with settings:
-      | path      | folderMain |
-      | shareWith | Brian      |
-      | role      | viewer     |
-    And user "Brian" has declined share "/Shares/folderMain" offered by user "Alice"
+    And user "Alice" has sent the following share invitation:
+      | resource        | folderMain |
+      | space           | project101 |
+      | sharee          | Brian      |
+      | shareType       | user       |
+      | permissionsRole | Viewer     |
+    And user "Brian" has disabled sync of last shared resource
```

Comment:
or by using existing available step
```suggestion
    And user "Brian" has disabled the auto-sync share
    And user "Alice" has sent the following share invitation:
      | resource        | folderMain |
      | space           | project101 |
      | sharee          | Brian      |
      | shareType       | user       |
      | permissionsRole | Viewer     |
```

Reply:
but this is also good
>And user "Brian" has disabled sync of last shared resource

Reply:
> or by using existing available step

This flow will not meet the expected test case.